src/pykiso/can_message.py

```diff
@@ -0,0 +1,28 @@
+##########################################################################
+# Copyright (c) 2010-2022 Robert Bosch GmbH
+# This program and the accompanying materials are made available under the
+# terms of the Eclipse Public License 2.0 which is available at
+# http://www.eclipse.org/legal/epl-2.0.
+#
+# SPDX-License-Identifier: EPL-2.0
+##########################################################################
+
+"""
+pykiso Control Message Protocol
+*******************************
+
+:module: can_message
+
+:synopsis: Message transfer by can
+
+.. currentmodule:: message
+"""
+
+from typing import Any
+
+
+class CanMessage():
+    def __init__(self, name: str, signals: dict[str, Any], time_stamp: float) -> None:
```

Comment:
We are supposed to be compatible with python 3.7+, so using builtin types as type hints isn't possible

Use the ones from typing instead:
```
from typing import Any, Dict

Dict[str, Any]
```

Reply:
Ok, i will fix it